src/utils/parse-query.js

```diff
@@ -35,4 +36,36 @@ const parseQuery = (query) => {
   return parsedQuery;
 };
 
-export { parseQuery };
+/**
+ * Generate a WHERE clause based on parsed filters
+ *
+ * @param {object} parsedFilters Parsed filter parameters
+ * @returns {string} WHERE clause string
+ */
+const generateWhereClause = (parsedFilters) => {
+  const conditions = Object.keys(parsedFilters).map((key) => {
+    const filter = parsedFilters[key];
+
+    if (typeof filter === 'string') {
+      return `${key} = '${filter}'`;
+    } else if (typeof filter === 'object') {
```

Comment:
Since the logic here is a little not that straightforward, I would make a comment here and describe under what circumstances these cases will be executed. It would be helpful to provide a few simple examples.

Reply:
sure, commenting.

Reply:
done.